python/copr/v3/proxies/build.py

```diff
@@ -162,6 +162,9 @@ def check_before_build(self, ownername, projectname,
         :param buildopts: http://python-copr.readthedocs.io/en/latest/client_v3/build_options.html
         :return: Munch
         """
+        if project_dirname is None:
+            project_dirname = projectname
```

Comment:
I don't want to nitpick, but what happens if `project_dirname=None` goes down to the Frontend API code?  It seems we are hiding a Frontend API bug here ...